vmdb/app/models/vm_or_template/operations/power.rb

```diff
@@ -12,7 +12,7 @@ def raw_stop
   end
 
   def stop
-    raw_stop unless policy_prevented?(:request_vm_stop)
+    raw_stop unless policy_prevented?(:request_vm_poweroff)
```

Comment:
`policy_prevented?` will actually fire the event, thus a valid event name needs to be passed.